frontend/packages/client/src/components/Community/StrategyEditorModal/StrategyInformationForm.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { Input } from "components";
+
+const staticPlaceholders = {
+  contractAddress: "Contract Address",
+  contractName: "Contract Name",
+  maxWeight: "Max Weight",
+  minimunBalance: "Minimum Balance",
+};
+
+export default function StrategyInformationForm({
+  formFields = [],
+  formData = {},
+  setField = () => {},
+  actionButton,
+} = {}) {
+  return (
+    <>
+      <div className="columns is-flex-direction-column is-mobile m-0">
+        {formFields.map((field, index) => (
+          <Input
+            key={index}
+            placeholder={staticPlaceholders[field]}
+            name={field}
+            value={formData[field]}
+            onChange={setField(field)}
+          />
+        ))}
+      </div>
+      {actionButton}
+    </>
+  );
+}
```

Comment:
This is the form:
<img width="444" alt="Screen Shot 2022-06-17 at 21 04 08" src="https://user-images.githubusercontent.com/7526740/174414677-89150239-a4ee-4081-882e-cd1eaf6a4bbe.png">